docs/sources/enterprise/enterprise-configuration.md

```diff
@@ -326,3 +326,22 @@ A list of headers that are stripped from the outgoing data source and alerting r
 ### cookie_drop_list
 
 A list of cookies that are stripped from the outgoing data source and alerting requests.
+
+## [caching]
+
+> **Note:** Available in Grafana Enterprise v7.5 and later versions.
+> **Note:** Redis and memcached sections are in the defaults.ini setting, but redis and memcached backends are unavailable in Grafana Enterprise v7.5
```

Comment:
Is "redis" a proper noun? If so, please capitalize it.

Reply:
done